great_expectations/data_context/store/database_store_backend.py

```diff
@@ -242,10 +242,10 @@ def _get(self, key):
             .select_from(self._table)
             .where(
                 and_(
-                    *[
+                    *(
                         getattr(self._table.columns, key_col) == val
                         for key_col, val in zip(self.key_columns, key)
-                    ]
+                    )
                 )
             )
         )
```

Comment:
We see a few cases of this in the proposed changes; is there a case where converting this from a list to a generator causes us a problem? I would think it would only provide benefits (reduced memory usage) but I'm not confident.

Reply:
We are calling `and_()` with these *parameters so I do not see how *list comprehension vs. *generator expression will make a difference in populating the parameters other that the efficiency that you site.  If you see something that I do not then an issue to https://github.com/asottile/pyupgrade might be the best approach.